test/checks/readinessProbeMissing/success.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    test: readiness
+  name: readiness-exec
+spec:
+  containers:
+  - name: readiness
+    image: k8s.gcr.io/busybox
+    args:
+    - /bin/sh
+    - -c
+    - touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600
```

Comment:
The readiness probe tests that the pod is ready to go. So maybe instead doing the sleep 30 first then the touch and getting rid of the rm

Reply:
This actually comes straight from the k8s documentation 😄